docs/OVERVIEW.md

```diff
@@ -0,0 +1,111 @@
+# Dearborn Coding Club Website Front-end Documentation
+
+## Overview
+
+This document provides an overview of the front-end structure for the Dearborn Coding Club website. The application is built using React and React Router for navigation.
+
+## File Structure
+
+```
+src/
+├── main.tsx
+├── App.tsx
+├── pages/
+│   ├── Home.tsx
+│   └── Notes.tsx
+├── components/
+│   ├── Navbar.tsx
+│   ├── Footer.tsx
+│   └── Card.tsx
+├── contexts/
+│   └── ThemeContext.tsx
+└── css/
+    ├── main.css
+    ├── theme.css
+    ├── App.css
+    └── Home.css
+```
+
+## Component Hierarchy
+
+1. `main.tsx`: Entry point of the application
+2. `App.tsx`: Main component that sets up routing and layout
+   - `Navbar`: Navigation component
+   - `Routes`: Handles routing between pages
+     - `Home`: Home page component
+     - `Notes`: Notes page component
+   - `Footer`: Footer component
+
+## Routing
+
+The application uses React Router for navigation. Routes are defined in `App.tsx`:
+
+- `/`: Home page
+- `/notes`: Notes page
+
+## Key Components
+
+### App.tsx
+
+The main component that sets up the overall structure of the application. It includes:
+
+- `Navbar` component
+- `Routes` for handling navigation
+- `Footer` component
+
+### Home.tsx
+
+The home page component. It showcases:
+
+- Multiple `Card` components with information about the Dearborn Coding Club
+- Links to GitHub repositories and Slack organization
+
+### Notes.tsx
+
+A simple notes page with:
+
+- "Go Back" button for navigation
+- Placeholder text for notes functionality
+
+### ErrorPage.tsx
+
+A custom error page component that displays when routing errors occur.
+
+## Styling
+
+The application uses a combination of CSS files and inline styles:
+
+- `main.css`: Global styles
+- `theme.css`: Theme-related styles
+- `App.css`: Styles specific to the App component
+- `Home.css`: Styles specific to the Home component
+
+Some components (like `Notes.tsx` and `ErrorPage.tsx`) use inline styles for simplicity.
+
+## Context
+
+The application uses React Context for theme management:
+
+- `ThemeContext.tsx`: Provides theme-related functionality to the entire app
+
+## Contributing
+
+To contribute to the front-end:
```

Comment:
```suggestion
To contribute to the front-end, follow our [CONTRIBUTING.md](https://github.com/dearborn-coding-club/website-base-frontend/blob/main/CONTRIBUTING.md) guidance, as well as the following general rules:
```

Can we plug our [CONTRIBUTING.md](https://github.com/dearborn-coding-club/website-base-frontend/blob/main/CONTRIBUTING.md) file here?